src/stable/lib/globals.ts

```diff
@@ -12,42 +12,51 @@ import { Action } from './state';
 
 declare global {
     // eslint-disable-next-line no-var
-    var _azleActions: Action[];
+    var _azleActions: Action[] | undefined;
```

Comment:
src/experimental/build/commands/build/wasm_binary/rust/experimental_canister_template/src/internal_canister_methods.rs has _azleActions.length. I'm imagining that will fail with a can't read property length of undefined if _azleActions is undefined.